codebuild/codebuild.config

```diff
@@ -78,7 +78,49 @@ env: S2N_LIBCRYPTO=openssl-1.1.1 BUILD_S2N=true TESTS=valgrind GCC_VERSION=9
 snippet: UbuntuBoilerplate2XL
 env: S2N_LIBCRYPTO=openssl-1.0.2 BUILD_S2N=true TESTS=valgrind GCC_VERSION=6
 
-
 # SAW tests
+[CodeBuild:s2nSawHmacMd5]
+snippet: UbuntuBoilerplate2XL
+env: TESTS=sawHMAC SAW_HMAC_TEST=md5 SAW=true GCC_VERSION=NONE PLATFORM_CLANG=3.9
+
+[CodeBuild:s2nSawHmacSha1]
+snippet: UbuntuBoilerplate2XL
+env: TESTS=sawHMAC SAW_HMAC_TEST=sha1 SAW=true GCC_VERSION=NONE PLATFORM_CLANG=3.9
+
+[CodeBuild:s2nSawHmacSha224]
+snippet: UbuntuBoilerplate2XL
+env: TESTS=sawHMAC SAW_HMAC_TEST=sha224 SAW=true GCC_VERSION=NONE PLATFORM_CLANG=3.9
+
+[CodeBuild:s2nSawHmacSha256]
+snippet: UbuntuBoilerplate2XL
+env: TESTS=sawHMAC SAW_HMAC_TEST=sha256 SAW=true GCC_VERSION=NONE PLATFORM_CLANG=3.9
+
+[CodeBuild:s2nSawHmacsha384]
+snippet: UbuntuBoilerplate2XL
+env: TESTS=sawHMAC SAW_HMAC_TEST=sha384 SAW=true GCC_VERSION=NONE PLATFORM_CLANG=3.9
+
+[CodeBuild:s2nSawHmacSha512]
+snippet: UbuntuBoilerplate2XL
+env: TESTS=sawHMAC SAW_HMAC_TEST=sha512 SAW=true GCC_VERSION=NONE PLATFORM_CLANG=3.9
+
+[CodeBuild:s2nSawDrbg]
+snippet: UbuntuBoilerplate2XL
+env: TESTS=sawDRBG SAW=true GCC_VERSION=NONE PLATFORM_CLANG=3.9
+
+[CodeBuild:s2nSawSike]
+snippet: UbuntuBoilerplate2XL
+env: TESTS=sawSIKE_r1 SAW=true GCC_VERSION=NONE PLATFORM_CLANG=3.9
+
+[CodeBuild:s2nSawBike]
```

Comment:
Should sawBIKE be added to `s2n_codebuild.sh`?

Reply:
missed this, thanks